js/ui/map.js

```diff
@@ -169,9 +170,13 @@ class Map extends Camera {
 
         this.on('move', this._update.bind(this, false));
         this.on('zoom', this._update.bind(this, true));
+        this.on('movestart', () => {
+            this._isMoving = true;
+        });
         this.on('moveend', () => {
             this.animationLoop.set(300); // text fading
             this._rerender();
+            this._isMoving = false;
```

Comment:
So we should move `Map#isMoving` to `Camera#isMoving`? And add `Camera#moving` similarly to `Camera#rotating`, `Camera#pitching`, and `Camera#zooming`?